src/Commands/ConvertToDrupalRecommendedSiteCommand.php

```
@@ -187,6 +195,14 @@ private function updateComposerJsonMeta(string $localPath): void
         $composerJson['extra']['installer-paths']['web/profiles/custom/{$name}'] = ['type:drupal-custom-profile'];
         $composerJson['extra']['installer-paths']['web/themes/custom/{$name}'] = ['type:drupal-custom-theme'];
 
+        foreach (['autoload', 'scripts', 'scripts-description'] as $item) {
+            $composerJson[$item] = $composerManagedComposerJson[$item];
```

Is there a possibility for `$composerManagedComposerJson` not to have some keys?
In other words, should this be like: 
```
if (isset($composerManagedComposerJson[$item])) {
  $composerJson[$item] = $composerManagedComposerJson[$item];
}
```

---

I don't think so, it's here: https://github.com/greg-1-anderson/drupal-composer-managed/blob/default/composer.json

---

Seems safe enough like this

---

If we did add any special checking, I would put it in advance and fail fast, e.g. "drupal-composer-managed upstream missing expected portions of composer.json; can not continue, aborting." That might be prudent, and wouldn't require extra nesting in the loop.